docs/guides/admin/docs/configuration/security.jwt.md

```diff
@@ -12,6 +16,26 @@ This guide assumes that a JWT provider is already setup and Opencast receives th
 In order to integrate Opencast with an OIDC provider you could use the
 [oauth2-proxy](https://github.com/oauth2-proxy/oauth2-proxy).
 
+Enable Karaf Feature
+--------------------
+
+The JWT authentication and authorization is disabled by default and needs to be enabled before it can be used.
+For this, make sure the Karaf feature `opencast-security-jwt` boots when starting Opencast.
+
+For this, edit the configuration file `etc/org.apache.karaf.features.cfg` and add the feature:
+
+```
+  …
+  http-whiteboard/4.2.9, \
+  opencast-admin/11.0.0, \
+  opencast-security-jwt/11.0.0, \
```

Comment:
Here you use `11` but the PR is against the `10.x` branch. I don't think this is terribly important , especially since you mention using the correct version below.

Reply:
I've now made the version number dynamic, since you are the second one complaining about this.
I deliberately did not do that, so people realize that they need to adjust the number.
They will still need to set the minor version correctly on their own.